README.md

```diff
@@ -12,7 +12,7 @@ urlFragment: msal-android-app
 

```

Comment:
Is this README going to be displayed on msdocs or only here on GitHub? If only on GitHub, can we remove the meta-block in lines 1-10?

If this will be on the docs site, disregard this